.travis.yml

```diff
@@ -0,0 +1,13 @@
+dist: trusty
+language: cpp
+
+compiler:
+  - gcc
+  - clang
+  
+script:
+  - mkdir build && cd build
+  - cmake -DMATERIALX_BUILD_PYTHON=ON ..
+  - make -j4
```

Comment:
Ah, thanks for the heads-up!  We'll iterate on this number to find the optimal value for Travis.